### ISSUE TITLE
process.env.NODE_ENV is undefined when running eslint

### ISSUE THREAD

**Issue by philfontaine** [quasarframework/quasar#5095]
**Describe the bug**

process.env.NODE_ENV is always undefined when running eslint, be it with `quasar dev`, `quasar build` or `yarn lint`

**Codepen/jsFiddle/Codesandbox (required)**

The codesandbox provided does not have eslint integration, but it's very easy to create a local project using @quasar/cli

**To Reproduce**
1. `quasar create test-project`
2. Select all default options
3. Edit eslintrc.js, add the following at the beginning of the file:
 ```js
console.log(process.env.NODE_ENV);
console.log(); // These console.log are there so that the output is not overwritten by quasar
console.log();
console.log();
```
4. Run `quasar dev` or `quasar build` or `yarn lint`
5. `undefined` is outputted in the console

**Expected behavior**
process.env.NODE_ENV should be defined, as it is shown in the documentation and default project.

**Screenshots**
![image](https://user-images.githubusercontent.com/16233697/65161758-1d6f4080-da06-11e9-8fa4-08c47013afa6.png)

**Platform (please complete the following information):**
OS: Windows 10
Node: 10.16.1
NPM:
Yarn: 1.17.3
Browsers:
iOS:
Android:
Electron:

**Additional context**

**Comment by rstoenescu** [comment]
Hi,

The "process.env.<X>" variables are available in App's js context. Eslintrc file is run in Node context. Two different things.

**Comment by philfontaine** [comment]
I know, but why would you suggest using process.env.NODE_ENV in the eslintrc.js file in the [documentation](https://quasar.dev/quasar-cli/cli-documentation/linter#Lint-Rules) and include it in the default options of a quasar project if it does not even work : 
![image](https://user-images.githubusercontent.com/16233697/65165103-18ad8b00-da0c-11e9-85a6-6b8deef5ebed.png)

The user is not warned that it does not work, and is not provided with an alternative (e.g. you need to manually set the process.env.NODE_ENV variable before running quasar dev)

Are you seeing my point of view?

**Comment by rstoenescu** [comment]
Hi,

I now see the point. And the value in this.
That part was a remnant from long long ago. Pushed a fix for quasar/app and will be available in 1.1.1. Until it gets released, yes, please specify the env var yourself.

Thank you for insisting on this!

**Comment by W3Web** [comment]
Hi,

Is possible to auto remove console.log for production to avoid this error ?

```
ERROR in ./src-pwa/register-service-worker.js
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

D:\code\Janus\Janus-App\src-pwa\register-service-worker.js
  15:5  error  Unexpected console statement  no-console
  19:5  error  Unexpected console statement  no-console
  23:5  error  Unexpected console statement  no-console
  27:5  error  Unexpected console statement  no-console
  31:5  error  Unexpected console statement  no-console
  44:5  error  Unexpected console statement  no-console
  48:5  error  Unexpected console statement  no-console
```
